pymesync.py

```diff
@@ -18,11 +23,12 @@
 
 class TimeSync(object):
 
-    def __init__(self, baseurl, user, password, auth_type):
+    def __init__(self, baseurl):
         self.baseurl = baseurl
-        self.user = user
-        self.password = password
-        self.auth_type = auth_type
+        self.user = None
```

Comment:
If I want to check them later, it's easier to check if they are None than something else, like an empty string. I currently only check token, but may check other things in future revisions.